### ISSUE TITLE
Backdrop background showing after opening bottomSheetModal

### ISSUE THREAD

**Issue by andreibahachenka** [gorhom/react-native-scroll-bottom-sheet#845]
# Bug

Hey all!

The backdrop background appears after fully opening the bottom sheet modal
It happened after updating bottom-sheet to 4.1.5 version

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
 "@gorhom/bottom-sheet": "^4.1.5",
   "react-native": "0.64.3",
  "react-native-reanimated": "^2.3.1",
 "react-native-gesture-handler": "~2.1.0",

## Steps To Reproduce

I have attached the GIF to check, just click on the image below
https://im3.ezgif.com/tmp/ezgif-3-f8056682c5.gif

Describe what you expected to happen:

Background is showing only after bottomSheetModal opens

**Comment by i8ramin** [comment]
Hi, looks like this issue was closed b/c it does not follow issue template, but it is an actual bug. experiencing the same issue. The backdrop shows _after_ the modal appears. Ideally, it should show up before or at the same time

**Comment by i8ramin** [comment]
Created a new issue to track this here: https://github.com/gorhom/react-native-bottom-sheet/issues/857
